levels/ending/script.c

```diff
@@ -16,22 +16,26 @@
 #include "make_const_nonconst.h"
 #include "levels/ending/header.h"
 
+const LevelScript level_ending_entry_loop[] = {
+    SLEEP(/*frames*/ 1),
+    JUMP(level_ending_entry_loop), // (loop sleep 1 forever)
+};
+
 const LevelScript level_ending_entry[] = {
-    /*0*/ INIT_LEVEL(),
-    /*1*/ LOAD_LEVEL_DATA(ending),
-    /*4*/ ALLOC_LEVEL_POOL(),
-
-    /*5*/ AREA(/*index*/ 1, ending_geo_area_1),
-    /*7*/ END_AREA(),
-
-    /*8*/ FREE_LEVEL_POOL(),
-    /*9*/ SLEEP(/*frames*/ 60),
-    /*10*/ BLACKOUT(/*active*/ FALSE),
-    /*11*/ LOAD_AREA(/*area*/ 1),
-    /*12*/ TRANSITION(/*transType*/ WARP_TRANSITION_FADE_FROM_COLOR, /*time*/ 75, /*color*/ 0x00, 0x00, 0x00),
-    /*14*/ SLEEP(/*frames*/ 120),
-    /*15*/ CALL(/*arg*/ 0, /*func*/ lvl_play_the_end_screen_sound),
-    // L1:
-    /*17*/ SLEEP(/*frames*/ 1),
-    /*18*/ JUMP(level_ending_entry + 17), // goto L1 (loop sleep 1 forever)
+    INIT_LEVEL(),
```

Comment:
are these.... FORMATTING CHANGES??????

Reply:
They're somewhat important changes actually; the level script commands for `SET_BACKGROUND_MUSIC` are different sizes depending on whether or not BETTER_REVERB is enabled, which causes hardcoded offsets to be completely broken.